plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
+
+
+def generate_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--selector', type=str, action="store",
+                        required=True, help="Label Selector")
+    return parser
+
+
+check_mongodb_cmd = ("bash", "-c",
+                     "mongo 127.0.0.1/admin "
+                     "-u \"admin\" -p \"$MONGODB_ADMIN_PASSWORD\" "
+                     "--eval=\"rs.status().myState\" --quiet")
+
+
+def parse_mongo_result(output):
+    try:
+        return int(output.rstrip("\n"))
+    except:
+        return None
```

Comment:
What is `parse_mongo_result` intended to consume as argument, and what is it intended to return?

From the name, I guess the argument called "output" is a "mongo result", and probably a string, because we call a string method on it.

Now it's not clear what it returns, maybe a number, maybe None, though I don't know what it is semantically.

> Our current methodology for running exec in multiple pods in a single call to openshift.exec_in_pods hides our ability to track the individual return codes from each execution.

Perhaps we need to improve `openshift.exec_in_pods` to allow us do what we need?